.ci/docker/node-playwright/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN apt-get -qq update && apt-get -qq install -y curl && \
 RUN apt-get -qq install -y git --no-install-recommends
 
 # 3. Install Browser (Chrome, Firefox and Webkit) dependencies
-RUN apt-get -qq install -y  --no-install-recommends \
+RUN apt-get clean all -qq install -y  --no-install-recommends \
```

Comment:
The clean all has been added in order to avoid the following error:


<img width="1055" alt="Screenshot 2022-03-08 at 20 03 04" src="https://user-images.githubusercontent.com/15065076/157307278-04694ce3-0465-4d27-b525-3ef033e496fa.png">

That error was showing up in the CI after installing the new dependency.

That issue was not happening when executing in the two laptops used to debug this issue